### DISCUSSION TITLE
BT-10-Procedure-Buyer is associated to a tailored codelist but there is no parentId

### DISCUSSION THREAD

**Discussion by nithril** [OP-TED/eForms-SDK#771] [Q&A] [answered]
Hello,

The codelist definition associated to `BT-10-Procedure-Buyer` is 
```
"codeList" : {
      "value" : {
        "id" : "authority-activity",
        "type" : "flat"
      },
      "severity" : "ERROR"
    }
```

As `authority-activity` is actually a tailored code list of `main-activity` shouldn't the `codeList` clause contain a `parentId` field?

**Comment by YvesJo** [comment, marked as answer]
Hi,
In some cases BT-10 and BT-610 may coexist (when a Contracting Authority acts as a Contracting Entity). Both have an associated tailored list derived from main-activity. Lists for Contracting Authority and Contracting Entity differ. Field associated to both Business terms use a same XML construct differentiated by the listName attribute. Having the parentID specified would lead to have main-activity as listName value and there would no be a way to differentiate BT-10 from BT-610 in the XML when both coexist.

**Reply by nithril** [reply]
Thank you @YvesJo for your explanation.

I'm not very convinced by what prevent @listName to stay 'authority-activity' but still have a parentId on main-activity.

**Reply by YvesJo** [reply]
Hi,
If the Parent Id would be specified, then this should be used as listName attribute value and there would be no option to distinguish BT-10 from BT-610.
